scripts/tokenomics-emulation/network.ts

```diff
@@ -0,0 +1,158 @@
+import { mnemonicGenerate } from "@polkadot/util-crypto";
+import { ExtrinsicStatus } from "@polkadot/types/interfaces";
+import { EventRecord } from "@polkadot/types/interfaces";
+import { KeyringPair } from "@polkadot/keyring/types";
+import { KeypairType } from "@polkadot/util-crypto/types";
+import { ApiPromise, Keyring } from "@polkadot/api";
+import { WsProvider } from "@polkadot/api";
+import { formatBalance, u8aToHex } from "@polkadot/util";
+
+class Network {
+  private api: ApiPromise;
+  private keyRingType: KeypairType;
+  private senderAccount: KeyringPair;
+
+  constructor(private readonly config: any) {}
+
+  public async setupNetwork() {
```

Comment:
Let's not duplicate words. We already working with Network. Jus `setup` is enough.